### ISSUE TITLE
Feat: Add overview of genre's recently heard

### ISSUE THREAD

**Issue by chakrakan** [chakrakan/spotify-box#2]

**Comment by CodeF0x** [comment]
Hi @chakrakan! I recently modified your code to show the authenticated users most listened genres on a long term and short term basis. I could submit a merge request that implements this feature as I already have a solution for that, depending on how many genres you want to list?

[Output of my solution](https://github.com/CodeF0x/CodeF0x/blob/master/readme.md)

**Comment by chakrakan** [comment]
Hey @CodeF0x :)
 
I checked out your implementation and I'm happy to see that you've customized it as per your needs and have it update READMEs instead of gists as well! 

That was the primary intention for this repository and I've purposely left it pretty bare-bones with only one endpoint to minimize information overload and avoid clutter, so anyone can fork and modify adding as many endpoints as needed instead of removing features they don't need. 

I created this issue during the initial implementation and thought this would be a feature I would personally need. But since then, I've realized that gist preview windows are tiny on the main profile (as it is setup for me), not much info can be fit effectively so I didn't bother with this (also due a little laziness haha). 

I'll close this issue but, regardless, thanks for offering to help with the feature!
